mealie/core/settings/settings.py

```diff
@@ -236,7 +236,7 @@ def OPENAI_ENABLED(self) -> bool:
     # Testing Config
 
     TESTING: bool = False
-    model_config = SettingsConfigDict(arbitrary_types_allowed=True, extra="allow")
+    model_config = SettingsConfigDict(arbitrary_types_allowed=True, extra="allow", secrets_dir="/run/secrets")
```

Comment:
I haven't given thought to the full implications of this (or even looked at what SettingsConfigDict does), but I think we'd want this information reflected in the doco in some way.

Reply:
Done, thank you.